extensions/wikibase/module/scripts/warnings-renderer.js

```diff
@@ -85,8 +82,12 @@ WarningsRenderer._createFacetForWarning = function (warning) {
 };
 
 WarningsRenderer._renderWarning = function (warning, onLocateRows) {
-  var title = WarningsRenderer._replaceIssueProperties($.i18n('warnings-messages/' + warning.type + '/title'), warning.properties);
-  var body = WarningsRenderer._replaceIssueProperties($.i18n('warnings-messages/' + warning.type + '/body'), warning.properties);
+  var warningKey = warning.type;
+  if ( warning.properties && warning.properties.hasOwnProperty("item_entity") ) {
+       warningKey += "-with-suggested-value";
+       }
```

Comment:
This change should not be necessary. Passing an item inside a warning could be used for all sorts of purposes, not just for this particular example, so it's better to keep this generic. Instead, change the Scrutinizer class so that it emits a different warning id whenever it adds an `item_entity`.